src/vivarium_gates_lsff/results_processing/process_results.py

```diff
@@ -71,7 +71,15 @@ def read_data(path: Path, single_run: bool) -> (pd.DataFrame, List[str]):
         data[results.RANDOM_SEED_COLUMN] = data[results.RANDOM_SEED_COLUMN].astype(int)
         with (path.parent / 'keyspace.yaml').open() as f:
             keyspace = yaml.full_load(f)
-    return data, keyspace
+    years = read_model_spec_for_start_end(path.parent)
+    return data, keyspace, years
+
+
+def read_model_spec_for_start_end(path: Path) -> Tuple[int, int]:
+    with (path / 'model_specification.yaml').open() as f:
+        spec = yaml.full_load(f)
+    time_cfg = spec['configuration']['time']
+    return (time_cfg['start']['year'], time_cfg['end']['year']+1)
```

Comment:
I think it is more natural to add 1 when you doing the `range()` calculation in `build_results()`

Reply:
Done.